package.json

```diff
@@ -30,7 +30,6 @@
     "libraries/ui-library-react",
     "libraries/ui-library-angular",
     "examples/angular",
-    "examples/react",
```

Comment:
If we remove the react example, we should also remove the react example code and all links in the documentation.

Reply:
true, removed it now everywhere